README.md

```diff
@@ -386,6 +386,7 @@ function Form() {
 Please note that when `formState.setField` is called, any existing errors that might have been set due to previous interactions from the user will be cleared, and both of the `validity` and the `touched` states of the input will be set to `true`.
 
 It's also possible to clear a single input's value using `formState.clearField`.
+It's also possible to set the error value for a single input using `formState.setFieldError` and to clear a single input's value using `formState.clearField`.
```

Comment:
It looks like this is stated twice accidentally?

> to clear a single input's value using `formState.clearField`.

Reply:
Good catch 😅